src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-weight: 300;
+}
+
+.conteiner {
+  display: block;
+  margin: 20px 0 0;
+}
+
+.searchInp {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  /*margin: 20px 0 0;*/
```

Comment:
remove it

Reply:
if i remove it, search bar will be short

Reply:
and if i remove font-family, auto-test 5 failed